client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -168,9 +177,15 @@ const InstructionsRenderer = ({
     return (
         <>
             <NumberedList>{allInstructionsContent}</NumberedList>
+            {/* TODO: Remove 3 following lines to remove the Success Criteria once #863 is merged. Will need that
+                  functionality to show the commands and assertions specification table and if there are any
+                  exceptions, especially 0-level assertions */}
             <Heading>{pageContent.instructions.assertions.header}</Heading>
             {pageContent.instructions.assertions.description}
             <NumberedList>{assertionsContent}</NumberedList>
+            {settingsContent.length ? settingsContent : null}
+            {/* TODO: Show commands and assertions specification table with #863 is merged */}
+            {/*{commandAssertionsTable}*/}
```

Comment:
I think these comments can be removed. Otherwise they may cause conflicts if this is merged before https://github.com/w3c/aria-at-app/pull/882 is merged. Thanks to our issue tracking we know that we need to do this work and the comments may not help.

Reply:
Done.